### ISSUE TITLE
Weather Fog

### ISSUE THREAD

**Issue by Darakudemion** [OreCruncher/DynamicSurroundings#540]
So any kind of fog causes this to happen. Not sure what is causing this at all. 

https://i.imgur.com/QraHT3n.png

**Comment by OreCruncher** [comment]
Causing?  The pic leads me to believe that it is a weather effect in the nether done by Dynamic Surroundings.  Looks kinda like a dust storm.

**Comment by Darakudemion** [comment]
Is there anyway to make it so the mobs don't go completely black? Or at least change the opacity of the effect?

**Comment by OreCruncher** [comment]
No.  It is tied up in how fog and rendering works in 1.12.x.  Try disabling "Weather Fog" in the dsurround.cfg.  This will tell DS to omit calculating fog factors based on weather.

**Comment by Darakudemion** [comment]
It happens with anything the mod has to do with fog. :/

**Comment by OreCruncher** [comment]
Did you try disabling weather fog?

**Comment by Darakudemion** [comment]
Yes, the only thing that does anything is to just disable the mod from handling any fog at all, so just default fog only.
